File: app/src/main/java/br/com/pauloc/mercadin/DB/UsuarioContract.java
package br.com.pauloc.mercadin.DB;

import android.provider.BaseColumns;

public class UsuarioContract {
    private String TABELA_USUARIO = "usuario";

    static final class UsuarioColumns implements BaseColumns {
        static String EMAIL = "email";
        static String SENHA = "senha";
        static String LOGADO = "logado";
    }
}

File: app/src/main/java/br/com/pauloc/mercadin/DB/UsuarioContract.kt
package br.com.pauloc.mercadin.DB

import android.provider.BaseColumns

class UsuarioContract {
    private val TABELA_USUARIO = "usuario"

    internal object UsuarioColumns : BaseColumns {
        var EMAIL = "email"
        var SENHA = "senha"
        var LOGADO = "logado"
    }
}